rust/connlib/tunnel/src/device_channel/tun_windows.rs

```diff
@@ -82,17 +77,6 @@ impl Tun {
         let adapter = &Adapter::create(&wintun, ADAPTER_NAME, TUNNEL_NAME, Some(uuid))?;
         let iface_idx = adapter.get_adapter_index()?;
 
-        // Remove any routes that were previously associated with us
-        // TODO: Pick a more elegant way to do this
-        Command::new("powershell")
-            .creation_flags(CREATE_NO_WINDOW)
-            .arg("-Command")
-            .arg(format!(
-                "Remove-NetRoute -InterfaceIndex {iface_idx} -Confirm:$false"
```

Comment:
This didn't seem to do anything at all. I think Windows already drops the routes when the tunnel goes down, but I'm not sure how to prove that.